main.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/signalfx/golib/datapoint"
+	"github.com/signalfx/golib/sfxclient"
+
+	"gopkg.in/Clever/kayvee-go.v6/logger"
+
+	"github.com/Clever/fluentd-kinesis-forwarder-monitor/config"
+)
+
+var log = logger.New("fluentd-kinesis-forwarder-monitor")
+var sfxSink = sfxclient.NewHTTPSink()
+
+type posInfo struct {
+	logFile string
+	inode   uint64
+	offset  int64
+}
+
+func sendToSignalFX(timestamp time.Time) error {
+	points := []*datapoint.Datapoint{}
+	dimensions := map[string]string{
+		"hostname": config.HOSTNAME,
+		"scope":    config.ENV_SCOPE,
+	}
+
+	datum := sfxclient.Gauge("fluentd-kinesis-forwarder-monitor", dimensions, timestamp.Unix())
+	points = append(points, datum)
+
+	return sfxSink.AddDatapoints(context.Background(), points)
+}
+
+func main() {
+	config.Initialize()
+
+	sfxSink.AuthToken = config.SIGNALFX_API_KEY
+
+	for {
+		ts, context, err := trackTimestamp(config.LOG_FILE_POS)
+		if err != nil {
+			log.ErrorD("track-timestamp", logger.M{"msg": err.Error()})
+		} else {
+			log.GaugeIntD("track-timestamp", int(ts.UnixNano()), logger.M{
+				"latest-log-ts": ts.String(), "context": context, "val-units": "nsec",
+			})
+
+			err = sendToSignalFX(ts)
+			if err != nil {
+				log.ErrorD("send-to-signalfx", logger.M{"msg": err.Error()})
+			}
+		}
+
+		time.Sleep(10 * time.Second)
+	}
+}
+
+func readLine(path string, start int64) (string, error) {
+	file, err := os.Open(path)
+	if err != nil {
+		return "", err
+	}
+
+	if _, err := file.Seek(start, 0); err != nil {
+		return "", err
+	}
+
+	r := bufio.NewReader(file)
+	data, err := r.ReadBytes('\n')
```

Comment:
It looks like the delimiter is always included: https://golang.org/pkg/bufio/#Reader.ReadBytes